README.md

```diff
@@ -17,6 +17,7 @@
 `MemoWise` is **the wise choice for Ruby memoization**, featuring:
 
   * Fast performance of memoized reads (with [benchmarks](#benchmarks))
+  * Strong [thread-safety guarantees](#thread-safety)
```

Comment:
Open to changing the order here!

Reply:
Might we consider "Documented and tested thread-safety guarantees" as an alternative to "Strong"? In thread-safety terminology, I think "Strong" may imply to some people that we have our own mutexes for example

Reply:
Yeah I wondered if "strong" would have specific meanings here. Will change!

Reply:
I also might have placed this at the bottom (under the API aspects), as I don't believe thread-safety guarantees being spelled out and tested is the very 2nd thing most are looking for?